core-web/libs/portlets/edit-ema/portlet/src/lib/shared/models.ts

```diff
@@ -208,7 +203,13 @@ export interface EditEmaDialogState {
     url: string;
     type: DialogType;
     payload?: ActionPayload;
-    editContentForm: DialogForm;
+    form: DialogForm;
+    clientAction: CLIENT_ACTIONS;
+}
+
+export interface DialogAction
+    extends Pick<EditEmaDialogState, 'payload' | 'form' | 'clientAction'> {
```

Comment:
What is `payload`? can we make it more clear?

Reply:
renamed